### PR TITLE
WebUI: Issue 3164

### LINKED ISSUE #3164: WebUI: Keytracking Mouse-Edit range is not similar to other parameters

### PR DESCRIPTION
**Pull request by schlenkibus**
closes #3164

### CODE REVIEW

File: projects/web/static/nonmaps/src/main/java/com/nonlinearlabs/client/world/maps/parameters/CombFilter/CombFilterCol2.java [hunk 1]
```diff
@@ -50,6 +50,6 @@ public Control doubleClick(Position pos) {
 		super(parent);
 		addChild(new Pitch(this));
 		addChild(new ModulatableHorizontalSlider(this, 118));
-		addChild(new ModulationSourceSwitch(this, 117));
+		addChild(new ModulationSourceKeytracking(this, 117));
```
Comment (schlenkibus) [inline projects/web/static/nonmaps/src/main/java/com/nonlinearlabs/client/world/maps/parameters/CombFilter/CombFilterCol2.java:53]: Naming is misleading

File: projects/web/static/nonmaps/src/main/java/com/nonlinearlabs/client/world/maps/parameters/OscA/FluctColumn.java [hunk 2]
```diff
@@ -4,7 +4,7 @@
 import com.nonlinearlabs.client.world.maps.parameters.ModulatableHorizontalSlider;
 import com.nonlinearlabs.client.world.maps.parameters.ModulateableKnob;
 import com.nonlinearlabs.client.world.maps.parameters.ModulationSourceSlider;
-import com.nonlinearlabs.client.world.maps.parameters.ModulationSourceSwitch;
+import com.nonlinearlabs.client.world.maps.parameters.ModulationSourceKeytracking;
```
Comment (schlenkibus) [inline projects/web/static/nonmaps/src/main/java/com/nonlinearlabs/client/world/maps/parameters/OscA/FluctColumn.java:7]: unused import

Comment (schlenkibus) [inline projects/web/static/nonmaps/src/main/java/com/nonlinearlabs/client/world/maps/parameters/OscA/FluctColumn.java:7]: unused import

File: projects/web/static/nonmaps/src/main/java/com/nonlinearlabs/client/world/maps/parameters/CombFilter/CombFilterCol2.java (hunk 1 above)
Comment (MatthiasSeeber) [reply projects/web/static/nonmaps/src/main/java/com/nonlinearlabs/client/world/maps/parameters/CombFilter/CombFilterCol2.java:53]: proposal: `UnmodulateableNumericDisplay` seems to be a more fitting name

File: projects/web/static/nonmaps/src/main/java/com/nonlinearlabs/client/world/maps/parameters/UnmodulateableNumericDisplay.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
+package com.nonlinearlabs.client.world.maps.parameters;
+
+import com.nonlinearlabs.client.world.maps.MapsLayout;
+
+public class UnmodulateableNumericDisplay extends ModulationSourceHighPriority {
```
Comment (MatthiasSeeber) [inline projects/web/static/nonmaps/src/main/java/com/nonlinearlabs/client/world/maps/parameters/UnmodulateableNumericDisplay.java:5]: It seems that the misleading naming goes further into the class hierarchy..(maybe worth a separate, non-prioritized ticket?)

**Review by MatthiasSeeber — APPROVED** [review]

**Pull request merged**